codecs/jdk/src/test/java/com/datastax/oss/dsbulk/codecs/jdk/collection/ListToUDTCodecTest.java

```diff
@@ -65,7 +65,7 @@ void should_convert_from_valid_internal() {
         .convertsFromInternal(udt1Value)
         .toExternal(stringList)
         .convertsFromInternal(udt1.newValue())
-        .toExternal(Arrays.asList("", ""))
+        .toExternal(Arrays.asList(null, null))
```

Comment:
shouldn't we keep the old tests as well?
To test the new functionality, but also handling of `""`?

Reply:
There is no way anymore by which you could transform an (empty) instance of `udt1` into an empty string. A null filed will always be translated into a null list element, and vice versa.